frontend/tests/components/Header.test.tsx

```diff
@@ -82,15 +82,6 @@ describe("Header", () => {
     expect(homeLink).toHaveAttribute("href", "/");
   });
 
-  it("display text without a home link if on home page", () => {
-    usePathnameMock.mockReturnValue("/");
-    render(<Header />);
-
-    const homeText = screen.getByText("Simpler.Grants.gov");
-    expect(homeText).toBeInTheDocument();
-    expect(homeText).not.toHaveAttribute("href", "/");
-  });
-
```

Comment:
After discussing w/ @doug-s-nava, I nixed this test. A couple reasons: The change from text to logo broke the test and it didn't seem worth reimplementing. The test was already kinda not perfect. It checked that the logo text doesn't have a link to the homepage when you're on the homepage. But that was only true after navigating to another page and returning to the homepage. A direct load/refresh of the homepage did have a link to the homepage. And the home menu link always has a link to the homepage.

Reply:
cc @acouch I think you wrote this test. Any concern w/ removing it?